documentation/examples/prometheus.yml

```diff
@@ -20,7 +20,7 @@ rule_files:
 # Here it's Prometheus itself.
 scrape_configs:
   # The job name is added as a label `job=<job_name>` to any timeseries scraped from this config.
-  - job_name: "prometheus"
+  - job_name: "extended-ceph-exporter"
```

Comment:
I still see this unrelated change in here?

Reply:
Fixed, all unrelated changes should be gone now